sympy/matrices/tests/test_commonmatrix.py

```diff
@@ -1340,3 +1340,8 @@ def test_limit():
     x, y = symbols('x y')
     m = CalculusOnlyMatrix(2, 1, [1/x, y])
     assert m.limit(x, 5) == Matrix(2, 1, [S(1)/5, y])
+
+def test_issue_13774():
+    M = Matrix([[1, 2, 3], [4, 5, 6], [7, 8, 9]])
+    v = [1,1,1]
+    raises(TypeError, lambda: M*v)
```

Comment:
Also add a test for v*M

Reply:
for v*M again same thing is happening which was previously happening for M*v. Now it seems scalar multiplication error. Where in code should I fix it ?

Reply:
From where in code this is happening ?
```
>>> 8*A
Matrix([
[ 8, 16],
[24, 32]])
>>> [1,2,'r']*A
Matrix([
[                  [1, 2, r],                   [1, 2, r, 1, 2, r]],
[[1, 2, r, 1, 2, r, 1, 2, r], [1, 2, r, 1, 2, r, 1, 2, r, 1, 2, r]]])
```

Reply:
Ping @asmeurer

Reply:
Please also add a test for `v*M`.